### DISCUSSION TITLE
Feature Request: Can a PDF for Flack2.0 Docs be created on: flask.palletsprojects.com webpage?

### DISCUSSION THREAD

**Discussion by animehunter123** [pallets/flask#4345] [Q&A] [answered]
Hello,

Thank you for your great work over the years in flask.

I noticed that on https://flask.palletsprojects.com/en/2.0.x/ <-- you cannot download the docs as a PDF file, where 1.0.x could (when you click the bottom right icon).

Can a pdf file please be added for the latest flask docs? This would help users who study flask offline.

Thanks! Here is a photo from my webbrowser:

![flask2doesnthavepdflink](https://user-images.githubusercontent.com/42163211/142199292-11bdfde1-73f4-429d-b5e3-50df661d4ffd.png)

**Comment by Yourun-proger** [comment, marked as answer]
@animehunter123 , hi! There was the following [discussion](https://github.com/pallets/flask/issues/4231) about this, where it says that the documentation will no longer be updated. But you can do it manually using [this](https://flask.palletsprojects.com/en/2.0.x/contributing/#building-the-docs)
